src/index.ts

```diff
@@ -62,13 +62,13 @@ class Collection<K, V> extends Map<K, V> {
 	}
 
 	/**
-	 * Identical to [Map.has()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/has).
-	 * Checks if an element exists in the collection.
-	 * @param {*} key - The key of the element to check for
-	 * @returns {boolean} `true` if the element exists, `false` if it does not exist.
+	 * Based on [Map.has()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/has).
+	 * Checks if one or multiple elements exist in the collection.
+	 * @param {...*} keys - The keys of the elements to check for
+	 * @returns {boolean} `true` if the elements exist, `false` if they do not exist.
 	 */
-	public has(key: K): boolean {
-		return super.has(key);
+	public has(...keys: K[]): boolean {
+		return keys.every((k) => super.has(k));
```

Comment:
Do we have any performance implications here?

cc @kyranet

Reply:
**Short story**: most likely does, but chances are, it'll be very small.

**Long story**: I have been reading something about [V8's Adaptor Frame](https://v8.dev/blog/adaptor-frame), a blog that released just today and specifies how V8 handles the lack or excess of parameters. From a quick read, I think this does have a performance impact, but it might be small to notice (besides, there are other factors, such as [IC](https://mathiasbynens.be/notes/shapes-ics#ics) [which helps for the case of only one argument] and `Array#every` being optimised to be inlined) - I don't know how well they combine. However, the performance will improve in the future.

To be completely sure, we would need to benchmark this and compare the performance of the regular usage (one single argument) versus the extended (multiple arguments).